docs/UserGuide.md

```diff
@@ -114,20 +114,20 @@ This detailed guide will walk you through how to make the most of **EduManage**
+|   7   |       Note        |                   `nt/NOTE`                    | Can take any values except prefixes already reserved, and can be blank.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                     |
+|   8   |   Phone Number    |                `p/PHONE_NUMBER`                | Should only contain numbers, should be at least 3 digits long, and can be the same as emergency contact number. International phone number formats (e.g., length greater than 8, different starting digits) are allowed.                                                                                                                                                                                                                                                                                                                                                                                                                                    |
+|   9   |      Subject      |                  `s/SUBJECT`                   | Should be valid for the given level: <ul><li>**Lower Secondary** (Level year is `S1` or `S2`): `MATH`, `SCIENCE`, `PHYSICS`, `CHEMISTRY`, `BIOLOGY`, `LITERATURE`, `HISTORY`, `GEOGRAPHY`, `SOCIAL_STUDIES`, `ENGLISH`, `CHINESE`, `HIGHER_CHINESE`, `MALAY`, `HIGHER_MALAY`, `TAMIL`, `HIGHER_TAMIL`, `HINDI`.</li><li>**Upper Secondary** (Level year is `S3`, `S4` or `S5`): `MATH`, `A_MATH`, `E_MATH`, `PHYSICS`, `CHEMISTRY`, `BIOLOGY`, `COMBINED_SCIENCE`, `ACCOUNTING`, `LITERATURE`, `HISTORY`, `GEOGRAPHY`, `SOCIAL_STUDIES`, `MUSIC`, `ART`, `ENGLISH`, `CHINESE`, `HIGHER_CHINESE`, `MALAY`, `HIGHER_MALAY`, `TAMIL`, `HIGHER_TAMIL`, `HINDI`. |
+|  10   |   Task Deadline   |               `d/TASK_DEADLINE`                | Should be in the format `YYYY-MM-DD`, and should only start from today's date (according to your computer's local time).                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                    |
```

Comment:
and should be from today's date onwards, inclusive of today's date (based on computer's local time).

Follow the bottom